src/common-principles.md

```diff
@@ -726,120 +731,123 @@ the Principle defines the *order of precedence* of such metadata files contents.
 
 ### Examples
 
-Example 1: Demonstration of inheritance principle
+!!! example "Example 1: Demonstration of inheritance principle"
 
-<!-- This block generates a file tree.
-A guide for using macros can be found at
- https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
--->
-{{ MACROS___make_filetree_example(
+    <!-- This block generates a file tree.
+    A guide for using macros can be found at
+    https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+    -->
+    {{ MACROS___make_filetree_example(
+        {
+        "sub-01": {
+            "func": {
+                "sub-01_task-rest_acq-default_bold.nii.gz": "",
+                "sub-01_task-rest_acq-longtr_bold.nii.gz": "",
+                "sub-01_task-rest_acq-longtr_bold.json": "",
+                }
+            },
+        "task-rest_bold.json": "",
+        }
+    ) }}
```

Comment:
Does the rendering code respect indentations?

Reply:
yeah I want to check how the macros and PDF are affected by this.

Reply:
keeping as draft for now.

Reply:
hum...

not great so far:

![image](https://github.com/user-attachments/assets/11c97d14-f6ea-4347-b43c-148c0b1d10b8)

I think the macro is affecting this

Reply:
same in the PDF

![image](https://github.com/user-attachments/assets/d6806fbd-cce8-499b-9e8b-79796d51b253)

Reply:
https://github.com/fralau/mkdocs-macros-plugin/issues/240#issuecomment-2350897840

Reply:
Sweet!!!

Need me update our contributing doc about this.